apis/Google.Cloud.Spanner.V1/Google.Cloud.Spanner.V1/SessionPool.DatabaseStatistics.cs

```diff
@@ -91,9 +96,11 @@ public sealed class DatabaseStatistics
                 long readWriteTransactionRequests,
                 long readWriteTransactionRequestsPrewarmed,
                 bool healthy,
-                bool shutdown)
+                bool shutdown,
+                string databaseRole)
```

Comment:
With the new pool key class, I'd expect the constructor to take that, and store it in a field - the DatabaseName and DatabaseRole properties can then just delegate to that (e.g. `public string DatabaseName => key.DatabaseName;`). We probably don't need to expose the key as a public property here. (It does need to be in the public API so that the Data package can use it, but I don't expect users to need it much.)